spec/outputs/elasticsearch_spec.rb

```diff
@@ -900,6 +900,34 @@ def mock_actions_with_response(*resp)
         end
       end
     end
+
+    context "when using http protocol" do
+      context "when using ssl without cert verification" do
+        protocol = "http"
+        subject do
+          require "logstash/outputs/elasticsearch"
+          settings = {
+            "protocol" => protocol,
```

Comment:
why not hardcode `http` here if no other is tested, and it is even mentioned in the higher level context